static-analysis/teammates-eslint.yml

```diff
@@ -40,12 +40,12 @@ rules:
     # disallow duplicate case labels
     no-duplicate-case: "error"
 
-    # disallow empty block statements
-    no-empty: "error"
-
     # disallow empty character classes in regular expressions
     no-empty-character-class: "error"
 
+    # disallow empty block statements
```

Comment:
I upgraded ESLint to 3.11.1 and the ruleset needs to be updated. In some places I re-ordered the rules as found on [this page](http://eslint.org/docs/rules/). Sorry for the confusion.